HelpSource/Classes/Array.schelp

```diff
@@ -57,37 +57,47 @@ copymethod:: ArrayedCollection *series
 
 copymethod:: ArrayedCollection *iota
 
-copymethod:: SequenceableCollection *interpolation
+copymethod:: Collection *interpolation
 
-copymethod:: SequenceableCollection *rand
-
-copymethod:: SequenceableCollection *rand2
-
-copymethod:: SequenceableCollection *linrand
-
-copymethod:: SequenceableCollection *exprand
-
-copymethod:: SequenceableCollection *fib
+copymethod:: Collection *rand
 
+copymethod:: Collection *rand2
 
+copymethod:: Collection *linrand
 
+copymethod:: Collection *exprand
 
+copymethod:: Collection *fib
 
 InstanceMethods::
 
 copymethod:: ArrayedCollection -at
 
-copymethod:: ArrayedCollection -put
+method:: put
+Put item at index, replacing what is there.
+code::
+x=[1, 2,3 , 4, 5]
+x.put(rrand(0, 4), [33, 77, 99, 322, 555, 666, 999].choose)
```

Comment:
The randomness here seems unnecessary -- I think this would be clearer without it